glide.yaml

```diff
@@ -0,0 +1,20 @@
+package: github.com/travis-ci/collectd-vsphere
+homepage: https://github.com/travis-ci/collectd-vsphere
+license: MIT
+import:
+- package: collectd.org
+  version: ~0.2.0
+  subpackages:
+  - api
+  - network
+- package: github.com/urfave/cli
+  version: v2
+- package: github.com/vmware/govmomi
+  version: ~0.8.0
```

Comment:
I think the version number is technically for the `govc` binary inside the govmomi package, but it's the best we have…